libs/designsystem/src/lib/components/list/list-item/list-item.component.ts

```diff
@@ -15,7 +15,7 @@ import { ThemeColor } from '@kirbydesign/core';
 import { PlatformService } from '../../../helpers/platform.service';
 import { ListSwipeAction, ListSwipeDirection, ListSwipeEnd } from '../list-swipe-action.type';
 
-export type BoundaryClass = 'first' | 'last';
+export type BoundaryClass = 'first' | 'last' | 'first last';
```

Comment:
Just a quick thought. Could the logic for applying these classes be modified so whenever an item is both first and last, it just gets both the type 'first' and 'last' instead of 'first last'?

Reply:
I was not able to find a logic that would be able to apply 'first' and 'last' on one element without using 'first last' because of the method return type.